content/en/v3/admin/platforms/k3s/_index.md

```diff
@@ -129,6 +144,87 @@ Note: If you get the error `Error enabling kubernetes auth: Post "https://127.0.
 ### Jenkins X v3 installation
 
 - Generate a cluster git repository from the [jx3-k3s-vault](https://github.com/jx3-gitops-repositories/jx3-k3s-vault) template, by clicking [here](https://github.com/jx3-gitops-repositories/jx3-k3s-vault/generate)
+- Clone the generated repository and cd into the repository folder
+- Set up ingress and webhook
```

Comment:
Would be nice to use one central page for ngrok, as this may be used by multiple, notably minikube